apps/molgenis-components/src/components/layout/LayoutModal.vue

```diff
@@ -62,6 +63,7 @@ export default {
     },
     /** Shown as the title of the model */
     title: { type: String, default: "" },
+    isCloseButtonShown: { type: Boolean, default: true },
```

Comment:
what about the 'X' in the top right corner

Reply:
I don't know what 'x' you mean, but this is the 'x' in top right corner of the layoutmodal.

Reply:
indeed, applied a fix.